### DISCUSSION TITLE
网站突然提示被阻断了

### DISCUSSION THREAD

**Discussion by lixioabai** [natfrp/wiki#167] [使用问题]
域名已经备案了。
之前弄的个人博客网站，用来记录开发遇到的一些问题。通过FRP穿透成功了。
很久没登录过了。今天登录一看 网站被阻断了。
1.域名备案（通过了）
2.网站备案（这个需要么）?
3.不良信息，违法等（不存在）

所以现在是什么情况是因为我用了以前跟网站一起备案用的域名的关系么？

**Comment by tidymoon** [comment]
节点名？
有部分节点只能穿透https流量（即使你有备案

**Reply by tidymoon** [reply]
另外，只需要域名备案，网站备案不需要

**Reply by lixioabai** [reply]
换了好几个隧道。以前都能成功的隧道现在好像都不行了。我是用的HTTP协议，换了几个成都的 十堰的 能建站的我基本上都试了一次，我感觉我的域名被拉黑了。怎么能查到是否被拉黑。是因为什么原因拉黑的。（是不是我网站套用的论坛的模板导致的，因为之前我的网站备案的时候，论坛或者有群体聊功能的需要到公安局特殊备案）

**Reply by tidymoon** [reply]
论坛确实是需要公安备案的.......
请问方便提供下域名么？

**Reply by lixioabai** [reply]
mylby.cn

**Reply by fakeboboliu** [reply]
你好，

![图片](https://user-images.githubusercontent.com/7552030/189846083-97f453a5-2d2b-43e5-9e3b-b85871ab5278.png)

**Reply by lixioabai** [reply]
我好像明白了。我服务器到期了。然后阿里云请求注销服务器然后 又注销了网站备案。 所以我的域名备案也没了。
<img width="582" alt="微信截图_20220913161545" src="https://user-images.githubusercontent.com/38240337/189848813-d8ff5dad-aad0-4392-9257-1b7b2bb24122.png">

**Reply by tidymoon** [reply]
标准结局...
